start-ltm.ps1

```diff
@@ -67,13 +67,13 @@ if($help)
 $projectDir = $Env:PROJECTS_DIR
 if([string]::IsNullOrEmpty($projectDir))
 {
-    $projectDir = "E:\ESFA"
+    $projectDir = "C:\Code\Contracting\"
```

Comment:
Did you mean to commit these changes?

Reply:
I did not, it's usually stashed, I'll revert it to a previous version prior to my change